Makefile.work

```diff
@@ -246,12 +246,18 @@ endif
 
 endif
 
+SPLIT_LOG = | tee
+
+DOCKER_BASE_LOG = $(SLAVE_DIR)/$(SLAVE_BASE_IMAGE)_$(SLAVE_BASE_TAG).log
+DOCKER_LOG = $(SLAVE_DIR)/$(SLAVE_IMAGE)_$(SLAVE_TAG).log
```

Comment:
Could you please add the log files into the .gitignore?

Reply:
I'll add to gitignore, not sure this is appropriate for a target folder considering these dockers are not actual targets as far as I know.